### DISCUSSION TITLE
Github refused to connect

### DISCUSSION THREAD

**Discussion by caffe-ian** [community/community#65164] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm not sure what caused this, but the first time visiting github.com with my new laptop it works fine. The issue appears after I downloaded some files from github.

I've tried using other browsers and the issue persists, but I can still access the website on other devices on the same network. Also checked my Firewall and nothing is wrong with it. Checked my Windows Security and there's no blocked websites in the list. I can still access other websites such as [google.com](https://google.com/) and [twitter.com](https://twitter.com/),

I've tried using a VPN and it works completely fine, how do I resolve this?
I'm on Windows 11
![github](https://github.com/community/community/assets/78070496/b348464c-bb95-49d6-92ed-b7ef494013f7)

**Comment by arafajs** [comment]
If it works fine with a VPN. Then the problem related to your location and IP.

**Reply by caffe-ian** [reply]
Yeah I've noticed it too, but it works on other devices on the same network

**Reply by arafajs** [reply]
@Chock1 
Go to Menu, and `Disable Opera Turbo`. ( remove the tick ) → Should solve the issue.

**Reply by caffe-ian** [reply]
Couldn't seem to find the setting, I don't think it's caused by the browser because I've tried accessing github on Microsoft Edge and Chrome and the issue still persists

**Reply by arafajs** [reply]
- Make sure that your device's date and time are set correctly.
- Check for updates for your device's operating system and browsers.

**Comment by peeter7** [comment]
I am also facing this issue, Can anybody helps me, I want to set the iframe of th github codein my website page so that it will show the page after running the code.

**Comment by bleachfish** [comment]
Im also facing this problem ,I downloaded some files from github ,then I can't access to this web site on PC any longer, even if I used VPN

**Comment by caffe-ian** [comment, marked as answer]
It turns out my computer somehow blocked the website and IP in the hosts file, this is how I solved it
https://winaero.com/block-websites-using-hosts-file-in-windows-10/
